dashboard/lti_new.py

```diff
@@ -0,0 +1,198 @@
+import logging, string, random
```

Comment:
I'd like to break this up into 

generic lti code
myla lti (that extends this generic code) 

But we can leave it like this for now. 

I'd probably rename this to something like myla_lti_1p3.py

Reply:
I'm not enthusiastic about the version number in the module name.  However, I agree that it'd be good to separate generic and MyLA-specific code somewhat.  Do you mean different modules, or just the organization within this module?

Besides the pylit1p3 module, aren't there modules around that define important things like LTI role URIs?  Doesn't IMS Global make something like this available?

Reply:
Yeah it could probably be called lti_advantage rather than have this specific version. 

AFIAK IMS doesn't support any python modules aside from the Caliper one. 

In a perfect situation there probably would be
* A separate project (or at least module) that has all these LTI constants
* A separate project that just has the LTI Advantage specific code for Django (like we had for 1.1)

The author of the library hasn't done this yet. I think we could split this out to only need to be included as INSTALLED_APPS [like the 1.1 code was](https://github.com/ccnmtl/django-lti-provider) but I haven't seen anyone that did that. The project author just had an [example with everything together.](https://github.com/dmitry-viskov/pylti1.3-django-example)